sdk/java-sdk-protobuf/src/main/scala/kalix/javasdk/impl/MetadataImpl.scala

```diff
@@ -291,4 +302,17 @@ object MetadataImpl {
         throw new RuntimeException(s"Unknown metadata implementation: ${other.getClass}, cannot send")
     }
 
+  def of(entries: Seq[MetadataEntry]): MetadataImpl = {
+    val transformedEntries =
+      entries.map { entry =>
+        // is incoming ce key in one of the alternative formats?
+        // if so, convert key to our internal default key format
+        alternativeKeyFormats.get(entry.key) match {
+          case Some(defaultKey) => MetadataEntry(defaultKey, entry.value)
+          case _                => entry
+        }
+      }
```

Comment:
I changes this implementation. 
Before, I was replacing any key starting with `ce_` with a key starting with `ce-`. 

This new implementation creates a Map for all know keys,like `ce_id` maps to `ce-id`. If we receive a `ce_id`, we transform it. 

If we receive keys starting with `ce_`, but not in the list of known keys, we don't touch them.